src/com/lftechnology/dec31/EmployeeTest.java

```diff
@@ -0,0 +1,128 @@
+package com.lftechnology.dec31;
+
+/**
+ * 
+ * 5. Create a class called Employee that includes three pieces of information
+ * as instance variables a first name (type String), a last name (type String)
+ * and a monthly monthSalary (double). Your class should have a constructor that
+ * initializes the three instance variables. Provide a set and a get method for
+ * each instance variable. Write a test application named EmployeeTest that
+ * demonstrates class Employee's capabilities. Create two Employee objects and
+ * display each object's yearly monthSalary. Then give each Employee a 10% raise
+ * and display each Employee's yearly monthSalary again.
+ * 
+ * 
+ * This class uses object of class {@link Employee}. The set and get methods are
+ * used to retrieve and assign the variables, while display is used to show
+ * employee details.
+ * 
+ * @author bibhushan
+ * 
+ */
+public class EmployeeTest {
+
+	public static void main(String[] args) {
+		Employee e1 = new Employee();
+		Employee e2 = new Employee();
+		e1.setFirstName("Ram");
+		e1.setLastName("Sharma");
+		e1.setMonthSalary(100);
+		System.out
+				.println("First name\t Last name\t yearSalary\t raisedSalary");
+		e1.displayEmployeeDetail();
+		e2.setFirstName("Hari");
+		e2.setLastName("Sharma");
+		e2.setMonthSalary(200);
+		e2.displayEmployeeDetail();
+
+	}
+}
+
+/**
+ * This class includes variables first name (type String), last name (type
+ * String) and a monthly monthSalary (double). This has a constructor that
+ * initializes the three instance variables.The set and get method for each made
+ * for each variable. getRaisedSalary and displayEmployeeDetail are use to show
+ * employee information
+ * 
+ * @author bibhushan
+ * 
+ */
+class Employee {
+	private String firstName;
+	private String lastName;
+	private double monthSalary;
+
+	public Employee() {
+		firstName = "";
+		lastName = "";
+		monthSalary = 0;
+	}
+
```

Comment:
Should we include a default constructor also.It is set automatically by the compiler if there is no other constructor defined as in your case.